lib/model/frames/submission.js

```diff
@@ -90,7 +90,7 @@ Submission.Def = Frame.define(
   'localKey',                           'encDataAttachmentName',
   'signature',                          'createdAt',    readable,
   'instanceName', readable,             'instanceId',   readable,
-  'current',      readable,             'xml',
+  'current',                            'xml',
```

Comment:
Because this `current` field is important to knowing which version of a submission is the current one in `GET ...submissions/<instanceId>/versions`, I don't think we should make it un-readable here (which, as you've figured out, controls whether or not a field is returned in an API response).